Makefile

```diff
@@ -269,7 +269,7 @@ bin/bird6:
 	curl -sSf -L --retry 5 https://github.com/projectcalico/bird/releases/download/$(BIRD_VER)/bird6 -o $@
 	chmod +x $@
 
-bin/allocate-ipip-addr:
+bin/calico-node:
 	cp fakebinary $@
```

Comment:
consider updating `fakebinary` with reference to `calico-node --allocate-ipip-addr` - it currently references the no-longer-existing `allocate_ipip_addr` binary.